kinsokujiko.go

```diff
@@ -0,0 +1,26 @@
+package Kinsokujiko
+
+import (
+	"github.com/ikawaha/kagome/tokenizer"
+)
+
+type Master struct {
+	Sentence string
+}
+
+type Surface struct {
+	Surf string
+	Pos  string
+}
+
+type Surfaces []Surface
+
+func Tokenize(s Master) Surfaces {
```

Comment:
__[golint]__
* [golint] exported function Tokenize should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->